src/components/page-sidebar/sidebar-panel.scss

```diff
@@ -1,16 +1,16 @@
 @import "../vars.scss";
 
-.sidebarPanel {
+.sidebar-panel {
```

Comment:
Thanks for catching these naming convention issues!

Reply:
There's actually a lot more. I'll keep changing them as I come across them.